streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -371,6 +377,7 @@ public synchronized void flush() {
      * @throws ProcessorStateException if flushing failed because of any internal store exceptions
      */
     private void flushInternal() {
+        log.trace("Flushing internal store {}", name);
```

Comment:
Hmm, do we need this log entry here? Since `flushInternal` is only called by `flush`, which is called by the `StateStoreManager` which is already logged.